Cargo.toml

```diff
@@ -19,8 +19,10 @@ git = "https://github.com/gtk-rs/gdk"
 git = "https://github.com/gtk-rs/gtk"
 
 [features]
+#default = ["gtk_3_18"]
```

Comment:
What are we saying by including this line? It could be interpreted in unpredictable ways.